empire/server/modules/powershell/persistence/elevated/wmi_updater.py

```diff
@@ -75,19 +81,52 @@ def generate(main_menu, module: PydanticModule, params: Dict, obfuscate: bool =
         # built the command that will be triggered
         trigger_cmd = "$($Env:SystemRoot)\\System32\\WindowsPowerShell\\v1.0\\powershell.exe -NonI -W hidden -enc " + enc_script
         
-        if daily_time != '':
-            
+        if (daily_time != '' or day != '' or day_of_week != ''):
+
+            # add DailyTime to event filter
             parts = daily_time.split(":")
-            
+
             if len(parts) < 2:
-                return handle_error_message("[!] Please use HH:mm format for DailyTime")
```

Comment:
Wherever there is an error in generating the module code, instead of printing the error and returning an empty string, use the `handle_error_message` function which will still print the error message to the server but also makes it so the rest api (and the client/starkiller downstream) get the error message as well.

So
```
print(helpers.color("[!] Please use HH:mm format for DailyTime"))
return ""
```

becomes
```
return handle_error_message("[!] Please use HH:mm format for DailyTime")
```